### DISCUSSION TITLE
why aspnetcore nuget packages did not update ?

### DISCUSSION THREAD

**Discussion by JMPlus2015** [dotnet/aspnetcore#28876] [Q&A]
hi,
       I want to know why aspnetcore nuget packages did not update ?
        such as 
           Microsoft.AspNetCore.Mvc   2.2.0
          
        It's can't work with Net5.0.

       somebody can help ?  thanks.

**Comment by Tratcher** [comment]
See https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-5.0&tabs=visual-studio#remove-obsolete-package-references

**Comment by JMPlus2015** [comment]
sometimes want to use this package alone, not with a aspnetcore project ...
but in project some other referenced package are at higher version..
thanks

**Reply by Tratcher** [reply]
The alternative is a FrameworkReference:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore?view=aspnetcore-5.0&tabs=visual-studio#use-the-aspnet-core-shared-framework
